### ISSUE TITLE
Awesome extension! But how do we add new languages?

### ISSUE THREAD

**Issue by GwynethLlewelyn** [mirnovov/searchdocs#1]
I simply _adore_ your Nova plugin — so simple, yet it does everything I wanted! Now the big question is: short of having to fork your whole plugin every time I wish to add another language, wouldn't it be easier to have some sort of user preference to add further languages?

**Comment by mirnovov** [comment]
Good idea, apologies for the late response. Unfortunately, I don't have the time to add such a feature now, but I'll keep this issue open in case I come back to it, or in the unlikely event someone else decides to add it.

**Comment by GwynethLlewelyn** [comment]
That's more than fine — I only catch up with things occasionally — and it shouldn't be too hard to add such an option, so maybe I'll do your 'unlikely event'! 😸 No promises, though — for several personal reasons, I'm also running late on a project for a client, and there is little spare time for me to tackle this. But... we'll see. Sometimes inspiration (for that project) fails me, and I like to work for a few hours on completely unrelated things, just to keep those old neurons firing...
